### PR TITLE
add bounty bond and transaction fee

### LINKED ISSUE #9: Bounty Bond and Transaction Fee in Bounty Creation Screen
Dependent on #12 
The bond amount and transaction fee are displayed in DOT values, after the user enters details in the form.
The values will change depending on Bounty Title size and Bounty Value

Image

### PR DESCRIPTION
**Pull request by BlertaSunCacti**
Fixes #9

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 body {
   font-family: 'Overpass';
+  color: #28123E;
```
Comment (abdulmth) [inline src/app.css]: we should add colors to the file `tailwind.config.js` and reference them from there using something like: 
`background-color: theme(colors.gray.100);`

File: src/components/bounty-setup/BountyCreation.svelte [hunk 2]
```diff
@@ -24,6 +24,18 @@
 				<option value="alice">Alice</option>
 				<option value="bob">Bob</option>
 			</select>
+
+			<div class="deposit mt-5 mb-10 h-24">
```
Comment (abdulmth) [inline src/components/bounty-setup/BountyCreation.svelte]: where is the `deposit` class defined?

File: src/components/bounty-setup/BountyCreation.svelte [hunk 3]
```diff
@@ -38,6 +50,7 @@
 	}
 
 	.content {
+		background-color:rgba(155,133,177,0.2);
```
Comment (abdulmth) [inline src/components/bounty-setup/BountyCreation.svelte]: same thing here, colors is better defined in the tailwind config file.

**Review by abdulmth — CHANGES_REQUESTED** [review]

File: src/components/bounty-setup/BountyCreation.svelte (hunk 2 above)
Comment (BlertaSunCacti) [reply src/components/bounty-setup/BountyCreation.svelte]: Will remove it. It was a mistake.

**Review by abdulmth — APPROVED** [review]

**Pull request merged**